Procfile_docker

```diff
@@ -0,0 +1,4 @@
+web: bundle exec puma -C config/puma.rb -e development -p 3001
+webpacker: ./bin/webpack-dev-server # Live update webpack js
+log: tail -f log/development.log
+hard_worker: bundle exec sidekiq
```

Comment:
I'm getting an error when trying to use `rerun` gem inside a Docker container in Mac.

```
hard_worker.1 | /usr/local/bundle/gems/terminal-notifier-2.0.0/vendor/terminal-notifier/terminal-notifier.app/Contents/MacOS/terminal-notifier: 1: /usr/local/bundle/gems/terminal-notifier-2.0.0/vendor/terminal-notifier/terminal-notifier.app/Contents/MacOS/terminal-notifier: Syntax error: "(" unexpected
```

I could not understand the issue but for now, I just created another `Procfile` to be able to keep working.

I'll revisit this issue later and put more effort into trying to understand and solve it.

Reply:
The issue was `rerun` calling `terminal-notifier`.

It's only possible to use `terminal-notifier` on the host machine, not inside Docker. https://github.com/julienXX/terminal-notifier/issues/234

So passing `--no-notify` option solves the issue. 406ba5b3